packages/expo-cli/src/commands/push-creds-web.ts

```diff
@@ -0,0 +1,197 @@
+import * as ConfigUtils from '@expo/config';
+import { ApiV2, Exp, UserManager } from '@expo/xdl';
+import chalk from 'chalk';
+import { Command } from 'commander';
+import fs from 'fs-extra';
+
+import log from '../log';
+
+type VapidData = {
+  vapidSubject: string;
+  vapidPubkey?: string;
+  vapidPvtkey?: string;
+};
+
+const vapidSubjectDescription =
+  'URL or `mailto:` URL which provides a point of contact in case the push service needs to contact the message sender.';
+
+export default function (program: Command) {
```

Comment:
Looks like instead of removing the commands, they've currently been moved to this file? Or am I missing something?

Reply:
seems like an elegant way to just remove them without deleting altogether.